waiter/src/waiter/util/utils.clj

```diff
@@ -147,6 +146,9 @@
              (let [writer (OutputStreamWriter. (.getOutputStream resp))]
                (try
                  (json/write data-map writer :value-fn stringify-elements)
+                 (catch Exception e
+                   (log/error e "Exception creating streaming json response")
```

Comment:
Nitpick: it would be nice to add this in a separate PR

Reply:
Removed.

Reply:
#340